scala-package/examples/src/main/scala/org/apache/mxnetexamples/imclassification/util/PerformanceMonitor.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnetexamples.imclassification.util
+
+import java.io.{BufferedWriter, FileOutputStream, OutputStreamWriter}
+import java.lang.management.ManagementFactory
+import java.util.Date
+
+class PerformanceMonitor(filename: String) extends Runnable {
+
+  val bean = ManagementFactory.getOperatingSystemMXBean
```

Comment:
Yes, although I can't find a cleaner way than running nvidia-smi.  Following the recommendation from @lanking520, I am moving the PerformanceMonitor to a utility for the other examples to use as well and it will be in a later PR.